ui/src/notebooks/pipes/markdown/index.ts

```diff
@@ -170,7 +170,7 @@ register({
   component: MarkdownPanel,
   button: 'Markdown',
   initial: () => ({
-    text: TEST_MODE ? TESTCASE : '',
+    text: TEST_MODE ? TESTCASE : 'Type markdown here',
```

Comment:
do we wanna make this a markdown comment? `[//]: # (Write Markdown below)`

Reply:
I tried the syntax here and it appears as a link, not sure if there is a comment in markdown or nah